draft-dekater-scion-dataplane.md

```diff
@@ -1342,20 +1355,101 @@ This section describes the steps that a SCION egress border router must perform
 
 # Security Considerations
 
-TODO security considerations.
+This section describes the possible security risks and attacks that SCION's data plane may be prone to, and how these attacks may be mitigated. It first discusses security risks that pertain to path authorization, followed by a section on all other data-plane security risks.
+
+**Note:** This section only discusses SCION data plane- and forwarding-specific security considerations. For security considerations related to the SCION control-plane PKI, see {{I-D.scion-cppki}}. {{I-D.scion-cp}} includes security considerations that concern the SCION control plane and routing.
+
+
+## Security Risks in Regard to Path Authorization
+
+A central property of the SCION path-aware data plane is path authorization. Path authorization guarantees that data packets always traverse the network along path segments authorized in the control plane by all on-path ASes. This section discusses how an adversary may attempt to violate the path-authorization property, as well as SCION's answers to these attacks. The following attacks are imaginable:
+
+- Crafting unauthorized hop fields (see [](#crafting)),
+- modifying the validity period of a path segment (see [](#mod-validity)), or
+- path splicing, that is, constructing an unauthorized segment by combining hop fields of different valid segments (see [](#path-splicing)).
+
+
+### Crafting Unauthorized Hop Fields {#crafting}
+
+Hop fields are protected with MACs. To be able to craft and misuse unauthorized hop fields, an adversary must therefore either determine the MAC's key or crack the underlying MAC algorithm.
+
+To determine an unknown MAC key, the adversary may perform a brute-force attack. Candidate keys can be validated by checking the MAC contained in sample hop fields. But as SCION uses 128-bit keys by default, such an off-line attack is computationally infeasible in practice. Another obstacle to adversaries is the short life of the keys for MAC computation, with a validity period of 24 hours.
```

Comment:
IN this draft we don't say much about the AS forwarding key. We don't say that the key should be 128 bit, and we also don't explain what the assumptions and threat model is (e.g. that  the key is shared across all routers within an AS). We should clarify this.
Also, we don't describe how the key is rotated